database/mssql/scripts/dbscripts/PSP_PIMS_S17_00/Alter Up/01_PSP_PIMS_S16_01_Alter_Up.sql

```diff
@@ -0,0 +1,940 @@
+GO
+DROP TRIGGER [dbo].[PIMS_PERSON_A_S_IUD_TR]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop trigger dbo.PIMS_PIMPRV_A_S_IUD_TR
+PRINT N'Drop trigger dbo.PIMS_PIMPRV_A_S_IUD_TR'
+GO
+DROP TRIGGER [dbo].[PIMS_PIMPRV_A_S_IUD_TR]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop view dbo.PIMS_PROPERTY_BOUNDARY_VW
+PRINT N'Drop view dbo.PIMS_PROPERTY_BOUNDARY_VW'
+GO
+DROP VIEW [dbo].[PIMS_PROPERTY_BOUNDARY_VW]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop view dbo.PIMS_CONTACT_MGR_VW
+PRINT N'Drop view dbo.PIMS_CONTACT_MGR_VW'
+GO
+DROP VIEW [dbo].[PIMS_CONTACT_MGR_VW]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Drop primary key constraint dbo.INSPAY_PK
+PRINT N'Drop primary key constraint dbo.INSPAY_PK'
+GO
+ALTER TABLE [dbo].[PIMS_INSURANCE_PAYEE_TYPE]
+	DROP CONSTRAINT [INSPAY_PK]
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Alter table dbo.PIMS_LEASE
+PRINT N'Alter table dbo.PIMS_LEASE'
+GO
+ALTER TABLE [dbo].[PIMS_LEASE] ALTER COLUMN [LEASE_CATEGORY_TYPE_CODE] nvarchar(20) NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+ALTER TABLE [dbo].[PIMS_LEASE] ALTER COLUMN [LEASE_RESPONSIBILITY_TYPE_CODE] nvarchar(20) NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+ALTER TABLE [dbo].[PIMS_LEASE] ALTER COLUMN [LEASE_PMT_FREQ_TYPE_CODE] nvarchar(20) NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Alter table dbo.PIMS_PROPERTY_IMPROVEMENT
+PRINT N'Alter table dbo.PIMS_PROPERTY_IMPROVEMENT'
+GO
+ALTER TABLE [dbo].[PIMS_PROPERTY_IMPROVEMENT] ALTER COLUMN [IMPROVEMENT_DESCRIPTION] nvarchar(2000) NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+EXEC dbo.sp_rename N'dbo.PIMS_PROPERTY_IMPROVEMENT.UNIT' , N'ADDRESS', 'COLUMN'
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Alter table dbo.PIMS_PROPERTY_IMPROVEMENT_HIST
+PRINT N'Alter table dbo.PIMS_PROPERTY_IMPROVEMENT_HIST'
+GO
+ALTER TABLE [dbo].[PIMS_PROPERTY_IMPROVEMENT_HIST] ALTER COLUMN [IMPROVEMENT_DESCRIPTION] nvarchar(2000) NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+EXEC dbo.sp_rename N'dbo.PIMS_PROPERTY_IMPROVEMENT_HIST.UNIT' , N'ADDRESS', 'COLUMN'
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Alter table dbo.PIMS_PERSON
+PRINT N'Alter table dbo.PIMS_PERSON'
+GO
+ALTER TABLE [dbo].[PIMS_PERSON]
+	ADD [ADDRESS_COMMENT] nvarchar(2000) NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Alter table dbo.PIMS_PERSON_HIST
+PRINT N'Alter table dbo.PIMS_PERSON_HIST'
+GO
+ALTER TABLE [dbo].[PIMS_PERSON_HIST]
+	ADD [ADDRESS_COMMENT] nvarchar(2000) NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+-- Alter table dbo.PIMS_INSURANCE
+PRINT N'Alter table dbo.PIMS_INSURANCE'
+GO
+ALTER TABLE [dbo].[PIMS_INSURANCE] ALTER COLUMN [COVERAGE_LIMIT] money NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+ALTER TABLE [dbo].[PIMS_INSURANCE] ALTER COLUMN [EXPIRY_DATE] date NULL
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+ALTER TABLE [dbo].[PIMS_INSURANCE]
+	ADD [IS_INSURANCE_IN_PLACE] bit NULL CONSTRAINT [INSRNC_IS_INSURANCE_IN_PLACE_DEF] DEFAULT (CONVERT([bit],(0)))
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+ALTER TABLE [dbo].[PIMS_INSURANCE]
```

Comment:
@FuriousLlama does this break the insurance view page?

Reply:
It will for sure. Some of those fields are used when querying the insurance details, so after the entities are regenerated it will fail to compile. We need a task to fix that view, including compilation.